sdk/core/Azure.Core/src/ClientOptions.cs

```diff
@@ -90,6 +91,14 @@ public HttpPipelineTransport Transport
         /// </summary>
         public RetryOptions Retry { get; }
 
+        /// <summary>
+        /// The policy to use for retries. If a policy is specified, it will be used in place of the <see cref="Retry"/> property.
```

Comment:
```suggestion
        /// Gets or sets the policy to use for retries. If a policy is specified, it will be used in place of the <see cref="Retry"/> property.
```